storage/innobase/xtrabackup/test/t/bug1707918.sh

```diff
@@ -0,0 +1,27 @@
+
+master_id=1
+slave_id=2
+
+start_server_with_id $master_id
+load_sakila
+
+start_server_with_id $slave_id
+
+setup_slave $slave_id $master_id
+
+mkdir -p $topdir/untared_backup $topdir/backup
+
+xtrabackup --backup --slave-info --target-dir=$topdir/backup
+xtrabackup --backup --slave-info --stream=tar > $topdir/backup/xtrabackup.tar
```

Comment:
Please add one more - with `xbstream`. It was not affected by the bug, but still we will have better coverage.